tests/test_web/test_api_v1.py

```diff
@@ -884,6 +911,7 @@ def test_add_bundle_success(
 
     rv = client.post('/api/v1/builds/add', json=data, environ_base=auth_env)
     rv_json = rv.json
+    print(rv_json)
```

Comment:
Do we need this print in the test? I guess it was some leftover, right? xD

Reply:
Nice catch! It somehow slipped through.